### DISCUSSION TITLE
Empty popup on Auto-evaluation

### DISCUSSION THREAD

**Discussion by ghost** [chamilo/chamilo-lms#5878] [General]
**Describe**
I can't see comment when I take tests in auto-evaluation mode

**To Reproduce**
1) Using CPANEL, I installed CHAMILO 1.11.26.
2) I created a course.
3) I created a test.
4) I set up auto-evaluation.
5) I created a new question, adding answers and comments.
6) I tried to take the test, but when I click on the answers, the comments popup is empty.

Extra: The comments are correctly displayed at the end of the test and in the popup mode, but the popup mode is not suitable for my needs because doesn't allow to retry with an other answer

**Expected behavior**
See comments in auto-evaluation mode

**Screenshots**

![1](https://github.com/chamilo/chamilo-lms/assets/106649995/685b2826-e09e-4cca-8e1a-c5062a44d2d5)
![2](https://github.com/chamilo/chamilo-lms/assets/106649995/2fdc27c5-5372-4367-8fee-8abe7f8ac30c)
![3](https://github.com/chamilo/chamilo-lms/assets/106649995/9dc4b757-b560-4521-a6cd-ea3f33a47907)
![4](https://github.com/chamilo/chamilo-lms/assets/106649995/9343c31a-8bce-4db0-b7f9-c5db6d241496)
![5](https://github.com/chamilo/chamilo-lms/assets/106649995/0509a86d-01f0-468e-8b33-dd1d3af3e6db)
![6](https://github.com/chamilo/chamilo-lms/assets/106649995/4061e3e7-6978-42e2-b5fc-5cd3095e54c0)
![7](https://github.com/chamilo/chamilo-lms/assets/106649995/2011ef20-324d-4d10-984b-35d9f83500fd)

switching to popup mode comments are displayed properly

![8](https://github.com/chamilo/chamilo-lms/assets/106649995/2bc478f6-c3d0-4df4-bbd6-d98f9e76eddd)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser chrome 126.0.6478.127

**Server
- cPanel Version | 110.0 (build 34)
- Apache Version | 2.4.59
- MySQL Version | 10.6.18-MariaDB-cll-lve-log
- Architecture | x86_64
 - Version of Chamilo:  1.11.26
 - Version of PHP:  8.1

**Comment by Kaneda-1** [comment]
Hello @Redricck  you need to use PHP 7.4 instead of 8.1.
https://campus.chamilo.org/documentation/installation_guide.html

https://github.com/chamilo/chamilo-lms/releases

> "In the assets attached to this release, the default ones have been prepared to work with PHP 7.2 and above, while the ones marked -php74 have been specifically prepared to work on PHP no lower than 7.4.
> 
> Although we do not officially support PHP 8.0, 8.1 or 8.2 with the 1.11 branch of Chamilo, we tested it relatively thoroughly with the -php74 package and it worked well, to the minor exception of the edition of learning paths which is visually a bit broken.
> 
> We recommend using PHP 7.4 while we are running to deliver Chamilo 2.0 with full support for PHP 8.2 or superior."_

**Comment by ghost** [comment]
Thank you for your response. I tried changing the PHP version to 7.4, but the problem remains. However, I understand that you are focusing on developing version 2.
